server/src/test/java/com/sauron/service/CurrentBalanceServiceTest.java

```diff
@@ -0,0 +1,20 @@
+package com.sauron.service;
+
+import org.junit.Test;
+
+import java.math.BigDecimal;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class CurrentBalanceServiceTest {
+    private CurrentBalanceService currentBalanceService = new CurrentBalanceServiceImpl();
+    private static final BigDecimal HARDCODED_RETURN_VALUE = new BigDecimal("123.45");
+
+    @Test
+    public void mockedReturnValueShouldBeAsHardcoded() {
+        //given
+        //when
+        //then
+        assertThat(currentBalanceService.getCurrentBalance()).isEqualTo(HARDCODED_RETURN_VALUE);
```

Comment:
Since we use given-when-then convention, I would extract the result as variable and move the execution to `when` section. 
```
        //when
        BigDecimal result = currentBalanceService.getCurrentBalance()
        //then
        assertThat(result).isEqualTo(HARDCODED_RETURN_VALUE);
```

Reply:
Anyway, if we want to play with BDD, maybe we could consider using this library instead of writing comments with `given`, `when` and `then` (did I already mention that I don't like them?):
https://thepracticaldeveloper.com/2018/05/10/write-bdd-unit-tests-with-bddmockito-and-assertj/

Reply:
I think you mentioned it, yes. :)
If you get one more person to say this library is preferable, I'll update the code to use it.

Reply:
Other than the alternative library, suggested changes done.